src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Verbs.tt

```diff
@@ -103,11 +39,39 @@ foreach (XElement verb in commands.Descendants(XName.Get("Verb","")))
         {
             app.Command("<#= commandName #>", <#= commandFormalName #>Cmd =>
             {
+<#
+    // Look for options directly under the verb
+    foreach (XElement option in verb.Descendants())
+    {
+        if (option.Name == XName.Get("Noun"))
+        {
+            break;
+        }
+
+        var data = new OptionData(option);
+        if (data.BaseClass == "Argument")
+        {
+#>
+                CommandArgument <#= data.FormalName #> = <#= commandFormalName #>Cmd.Argument(
+                    "<#= data.LongNameOrName #>", <#= data.Help #>);
```

Comment:
```suggestion
                CommandArgument <#= data.FormalName #> = <#= commandFormalName #>Cmd.Argument(
                    "<#= data.LongNameOrName #>",
                    <#= data.Help #>,
                    multipleValues: <#= data.MultipleValues #>);
```

Adding support for multiple values under `Argument` element. [Reference](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PushCommand.cs#L66)

Reply:
Why don't you do a separate PR for this MultipleValues feature that you want to add for arguments (and other improvements you decide to make). You can do that as part of your using the feature.
If I add it to this PR, the feature isn't even being used yet, or tested in any way.
(this change looks good)